<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="accessibleName">
       <string/>
      </property>
      <attribute name="title">
       <string>Deployers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Deploy mods for:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deploy_for_box">
           <item>
            <property name="text">
             <string>All Deployers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Deployer</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>444</width>
            <height>492</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Fallout 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fo3_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fallout 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fo4_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Fallout 4 VR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fo4vr_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Fallout New Vegas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fonv_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Morrowind</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tes3_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Oblivion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tes4_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Skyrim</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tes5_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Skyrim SE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tes5se_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Skyrim VR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tes5vr_url_field"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Starfield</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="starfield_url_field"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>LOOT masterlist.yaml URLS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ask when...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remove_prof_cb">
         <property name="text">
          <string>removing a profle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remove_mod_cb">
         <property name="text">
          <string>removing a mod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remove_from_dep_cb">
         <property name="text">
          <string>removing a mod from a deployer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remove_bak_target_cb">
         <property name="text">
          <string>removing a backup target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remove_bak_cb">
         <property name="text">
          <string>removing a backup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>155</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Log Level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="log_level_box">
           <item>
            <property name="text">
             <string>Info</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Warning</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Error</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="show_warning_cb">
         <property name="text">
          <string>Show log on warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="show_error_cb">
         <property name="text">
          <string>Show log on error</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>171</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nexus_tab">
      <attribute name="title">
       <string>NexusMods</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="valid_api_key_label">
         <property name="text">
          <string>No valid API key set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="premium_user_label">
         <property name="text">
          <string>Account is not premium</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="api_key_label">
           <property name="text">
            <string>API Key: ***</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="show_api_key_button">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="set_api_key_button">
           <property name="text">
            <string>Set API key</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="change_api_pw_button">
           <property name="text">
            <string>Change Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>183</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
